recipes/scc/all/patches/2023.06-add-fmt-10-support.patch

```diff
@@ -0,0 +1,11 @@
+--- src/sysc/scc/trace/vcd_trace.hh
++++ src/sysc/scc/trace/vcd_trace.hh
+@@ -53,7 +53,7 @@
+ 
+ template<typename T>
+ inline void vcdEmitValueChangeReal(FPTR os, std::string const& handle, unsigned bits, T val){
+-    auto buf = fmt::format("r{:.16g} {}\n", val, handle);
++    auto buf = fmt::format("r{:.16g} {}\n", static_cast<double>(val), handle);
```

Comment:
This has been merged upstream in https://github.com/Minres/SystemC-Components/pull/51 and released in https://github.com/Minres/SystemC-Components/releases/tag/2024.03